rasa/core/brokers/pika.py

```diff
@@ -40,6 +46,22 @@ def initialise_pika_connection(
 
     import pika
 
+    parameters = _get_pika_parameters(
+        host, username, password, port, connection_attempts, retry_delay_in_seconds
+    )
+    return pika.BlockingConnection(parameters)
+
+
+def _get_pika_parameters(
```

Comment:
this needs a docstring and shouldn't be private (it's not a class member and is used from within the broker class)

Reply:
I made it private since it shouldn't be outside of the module.

Reply:
@ricwo is that ok or should I make it public?

Reply:
okay, fine to leave it private